src/main/java/org/team1540/robot2025/subsystems/grabber/GrabberIOTalonFX.java

```diff
@@ -0,0 +1,80 @@
+package org.team1540.robot2025.subsystems.grabber;
+
+import static org.team1540.robot2025.subsystems.grabber.GrabberConstants.*;
+
+import com.ctre.phoenix6.BaseStatusSignal;
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.CANdiConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.hardware.CANdi;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.units.measure.*;
+
+public class GrabberIOTalonFX implements GrabberIO {
+    private final TalonFX motor = new TalonFX(GRABBER_ID);
+    // TODO: Fix candi configuration stuff
+    private final CANdi beforeBreak = new CANdi(BEFORE_BREAK_CANDI_ID, CANDI_BUS);
+    private final CANdi afterBreak = new CANdi(AFTER_BREAK_CANDI_ID, CANDI_BUS);
```

Comment:
You will have to properly configure the CANDi digital inputs in the constructor according to the docs [here](https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/configs/CANdiConfiguration.html).